frontend/apps/reader/modules/readertypeset.lua

```diff
@@ -506,7 +506,7 @@ function ReaderTypeset:onSetPageMargins(margins, refresh_callback)
     if self.view.footer.has_no_mode or self.view.footer.reclaim_height then
         bottom = Screen:scaleBySize(margins[4])
     else
-        bottom = Screen:scaleBySize(margins[4] + DMINIBAR_HEIGHT)
+        bottom = Screen:scaleBySize(margins[4] + self.view.footer:getHeight())
```

Comment:
Shouldn't the `self.view.footer:getHeight()` be moved *outside* of the `scaleBySize`? Since I assume it's returning an already-scaled container size in pixels, right?

Reply:
Definitely, yeah.

Reply:
Indeed, it's made out from `height = Screen:scaleBySize(DMINIBAR_CONTAINER_HEIGHT)`.
After this is fixed, lgtm.